### PR TITLE
Embedded Monitor: improved bar animation

### PR DESCRIPTION
**Pull request by FloVanGH**
Should be much better now. Tweaking in a specific direction (easing, duration) should be easy now if necessary.

### CODE REVIEW

File: examples/energy-monitor/ui/widgets/bar_tiles.slint [hunk 1]
```diff
@@ -129,10 +129,10 @@ export component BarTiles {
 
             states [
                 active when active : {
-                    max: display-max;
+                    max: min(display-max);
```
Comment (ogoffart) [inline examples/energy-monitor/ui/widgets/bar_tiles.slint:132]: looks suspicious.  min usually takes several values.

Comment (FloVanGH) [reply examples/energy-monitor/ui/widgets/bar_tiles.slint]: done

File: examples/energy-monitor/ui/widgets/bar_tiles.slint [hunk 2]
```diff
@@ -42,7 +42,7 @@ component ValueLabel {
 component BarTile {
     in property <string> title <=> i-title.text;
     in property <image> icon <=> i-icon.source;
-    in property <int> max;
+    in property <int> max: min;
```
[the remark this answers is not in the document]
Comment (FloVanGH) [reply examples/energy-monitor/ui/widgets/bar_tiles.slint:45]: Yeah you're right. Make totally sense. But the way, the other issue with the animation was that it always started by 0 and then takes to long.

[the remark this answers is not in the document]
Comment (FloVanGH) [reply examples/energy-monitor/ui/widgets/bar_tiles.slint]: done

File: examples/energy-monitor/ui/widgets/bar_tiles.slint [hunk 3]
```diff
@@ -94,7 +94,7 @@ component BarTile {
             }
 
             ValueLabel {
-                text: min;
+                text: floor(min);
```
Comment (ogoffart) [inline examples/energy-monitor/ui/widgets/bar_tiles.slint:97]: (i'd have used round, but whatever works)

**Review by ogoffart — APPROVED** [review]

**Pull request merged**